### ISSUE TITLE
World Ferns of 2020-09-25: test report 2020-09-28

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/testing#22]
_Feature_: improved crawler with new IDs. Tribes & subtribes (lot of unnamed OTUs) are excluded.

**Comment by mdoering** [comment]
> _Feature_: Tribes & subtribes (lot of unnamed OTUs) are excluded.

Is this really a feature? It alters the data, these clades exist for a reason. Would it not be better to filter them out in the assembly? Having `Tribus Amauropeltoid clade` sounds useful to me.

**Comment by yroskov** [comment]
When clades, OTUs, whatever else will be resolved and established as taxa of proper ranks, they are welcome in the CoL. Otherwise, all kind of OTUs should stay on taxonomist's "lab desk".

**Comment by mdoering** [comment]
Agree, but when publishing to ChecklistBank this is not COL s.str. Data in there should as much as possible represent the original source. Only when we assemble COL from the sources we should filter out unwanted taxa like clades. My point is we remove data in the wrong step too early.

**Comment by yroskov** [comment]
>we remove data in the wrong step too early. 
Absolutely not. Moreover, full set of data in its original view is available on the Hassler's website.
